### ISSUE TITLE
Put caret in a proper place in case of struct type completion

### ISSUE THREAD

**Issue by ignatov** [go-lang-plugin-org/go-lang-idea-plugin#1165]
```go
package main
import "sync"
func main() {
    wg := sync.WaitGroup{}<caret>
}
```

**Comment by ignatov** [comment]
Relates to #1157.

**Comment by dlsniper** [comment]
This looks fixed to me in 884e81a9a6997153515f639e1cfd3468c389a436

**Comment by dlsniper** [comment]
I'll close this as the caret looks in the place I'd expect it to be. Please reopen if needed.
